### PR TITLE
Demo2

### PR DESCRIPTION
**Pull request by ManektechBA**
Design changes added from Nishant

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 ### Configure the author name and email address to be used with your commits. Note that Git strips some characters (for example trailing periods) from user.name
 
 ```
-git config --global user.name "Sam Smith"
-git config --global user.email sam@example.com
+git config --global user.name "David"
+git config --global user.email david@manektech.com
```
Comment (meetshahmt) [inline README.md:7]: is this required ?

File: README.md [hunk 2]
```diff
@@ -25,4 +25,6 @@ git add*
 
 ## Git training, day 2
 
-### Thank you...!
\ No newline at end of file
+### Thank you...!
```
Comment (meetshahmt) [inline README.md:28]: very good...

**Review by meetshahmt — CHANGES_REQUESTED** [review]
please complete this asap !

File: README.md (hunk 1 above)
Comment (ManektechBA) [reply README.md:7]: Yes added for new credentials as old one are lost!

File: README.md (hunk 2 above)
Comment (ManektechBA) [reply README.md:28]: Thanks!

**Review by meetshahmt — APPROVED** [review]

**Pull request merged**